src/map/ai/controllers/mob_controller.h

```diff
@@ -25,6 +25,13 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 #include "controller.h"
 #include "entities/mobentity.h"
 
+enum class FollowType : uint8
+{
+    None,
+    Roam,
+    RunAway,
+};
```

Comment:
We usually have enums in all caps, i think

Reply:
That's what I've primarily (solely?) seen but I made this change after being requested by @zach2good above. Maybe I misunderstood the casing request though?

Reply:
I dunnow, tbh. Waitingon Zach then.

Reply:
This is correct for enum class, old style enums are in all caps